api/v1beta1/aerospikecluster_validating_webhook_test.go

```diff
@@ -0,0 +1,357 @@
+package v1beta1
+
+import (
+	"testing"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var aslog = logf.Log.WithName("Test validateNsConfUpdate")
+
```

Comment:
your tests are a bit difficult to read because your device names are hard to distinguish (/opt/dataXX/xvdYY). I suggest to use a simpler form /dev/nvme1.

Reply:
Okaay